src/pyinfraboxutils/coverage.py

```diff
@@ -232,5 +286,29 @@ def __create_markup(self, badge_dir):
         return doc
 
     def parse(self, badge_dir):
-        self.__convert_xml()
+        if(self.input == "/"):
```

Comment:
this is not correct, the badge_dir is used for something else, this should not be the input directory. you have to check if self.input is a directory or a file.

Reply:
Ok. Should be fix in next commit